From c7e33f7335dabd545fa4a77c3faaf8cb91723257 Mon Sep 17 00:00:00 2001
From: Catalin Horghidan <catalin.horghidan@nxp.com>
Date: Wed, 13 Mar 2019 09:57:14 +0200
Subject: [PATCH 664/706] arm64: dts: ls1028a: remove unused
 phy-connection-type property

Signed-off-by: Catalin Horghidan <catalin.horghidan@nxp.com>
(cherry picked from commit 515db34e36ad46e07f27c495f310248831d28d9d)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 3f7bc96f956e..1eb90aa2216c 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -703,7 +703,6 @@
 			};
 			enetc_port2: pci@0,2 {
 				reg = <0x000200 0 0 0 0>;
-				phy-connection-type = "internal";
 				fixed-link {
 					speed = <1000>;
 					full-duplex;
@@ -743,7 +742,6 @@
 				/* internal to-cpu ports */
 				port@4 {
 					reg = <4>;
-					phy-connection-type = "internal";
 					fixed-link {
 						speed = <1000>;
 						full-duplex;
@@ -752,7 +750,6 @@
 				port@5 {
 					reg = <5>;
 					cpu-ethernet = <&enetc_port3>;
-					phy-connection-type = "internal";
 					fixed-link {
 						speed = <1000>;
 						full-duplex;
@@ -762,7 +759,6 @@
 			};
 			enetc_port3: pci@0,6 {
 				reg = <0x000600 0 0 0 0>;
-				phy-connection-type = "internal";
 				fixed-link {
 					speed = <1000>;
 					full-duplex;
-- 
2.17.1

